backend/src/main/kotlin/no/bekk/services/MicrosoftService.kt

```diff
@@ -59,4 +61,32 @@ class MicrosoftService {
             json.decodeFromString(responseBody)
         return microsoftGraphGroupDisplayNameResponse.value.map { it.displayName.split("TEAM - ").last() }
     }
+
+    suspend fun fetchFunkRegMetadata(accessToken: String, team: String, teamId: String): Any {
+        val funkRegEndpoint = "https://frisk-backend.fly.dev/metadata"
+
+        try {
+            val response: HttpResponse = client.get(funkRegEndpoint) {
+                bearerAuth(accessToken)
+                accept(ContentType.Application.Json)
+                url {
+                    parameters.append("team", team)
+                    parameters.append("team_id", teamId)
```

Comment:
Usikker på hvordan dette gjøres, men vil tro det er noe slikt
```suggestion
                    parameters.append("key", "team")
                    parameters.append("value", teamId)
```